test/isBigInt.js

```diff
@@ -0,0 +1,77 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+import MAX_SAFE_INTEGER from '../src/internal/polyfills/Number.MAX_SAFE_INTEGER';
+import MIN_SAFE_INTEGER from '../src/internal/polyfills/Number.MIN_SAFE_INTEGER';
+import BigInt from './internal/polyfills/BigInt';
+import args from './shared/arguments';
+import Symbol from './shared/Symbol';
+
+describe('isBigInt', function () {
+  context('given a big integer value', function () {
```

Comment:
And here comes the trick. We only run test if native `BigInt` is available. We only need to `wrap` first two `context`s. The third `it` can always run because we're asserting for `false`.
```js
if (BigInt !== undefined) {
  context('given a big integer value', function () {
    specify('should return true', function () {
     ...assertions...
    });
  });

  context('given a non-big-integer value', function () {
    specify('should return false', function () {
      ...assertions...
    });
  });
}